### PR TITLE
Fix gitgraph branch continues after merge

### LINKED ISSUE #11981: Damaged graph
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes ([URL](https://try.gitea.io/ahaha/glu/graph))
  - [ ] No
  - [ ] Not relevant

## Description

We have damaged graph on some projects. This is very disturbing. Thanks.

### PR DESCRIPTION
**Pull request by zeripath**
After fixing the initial problem in #11981 another
problem has come to light...

Fix #11981 (part 2)

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

File: web_src/js/vendor/gitgraph.js [hunk 1]
```diff
@@ -284,7 +284,7 @@ export default function gitGraph(canvas, rawGraphList, config) {
           if ((breakIndex = findLineBreak(nextRow)) !== -1) {
             nextRow.splice(breakIndex, 1);
           }
-        }
+        }
```
Comment (techknowlogick) [inline web_src/js/vendor/gitgraph.js]:
```suggestion
        }
```

Comment (zeripath) [inline web_src/js/vendor/gitgraph.js]:
```suggestion
        }
```

File: web_src/js/vendor/gitgraph.js [hunk 2]
```diff
@@ -387,6 +391,10 @@ export default function gitGraph(canvas, rawGraphList, config) {
             break;
 
           case '|':
+            if (prevColomn
+              && prevColomn === '\\') {
```
Comment (silverwind) [inline web_src/js/vendor/gitgraph.js]:
```suggestion
            if (prevColomn && prevColomn === '\\') {
```

**Review by silverwind — APPROVED** [review]

**Review by mrsdizzie — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
Please send backport

**Pull request merged**